src/spikeinterface/extractors/mcsh5extractors.py

```diff
@@ -121,7 +124,8 @@ def openMCSH5File(filename, stream_id):
     Tick = info["Tick"][0] / 1e6
     exponent = info["Exponent"][0]
     convFact = info["ConversionFactor"][0]
-    gain = convFact.astype(float) * (10.0**exponent)
+    gain_uV = 1e6 * (convFact.astype(float) * (10.0**exponent))
+    offset_uV = 1e6 * (info["ADZero"].astype(float) * (10.0**exponent))
```

Comment:
Given this from the specification:
![image](https://github.com/SpikeInterface/spikeinterface/assets/6429509/a27033e4-1467-4940-800d-d24e7c9ad20b)


Shouldn't the offset be negative?

Reply:
Great.

Reply:
In our implementation the offset is after the gain.
So the offset should be divided by gain I think. Can you check ?